pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -1131,7 +1159,9 @@ public void writeBundleDataOnZooKeeper() {
         // Write the bundle data to metadata store.
         List<CompletableFuture<Void>> futures = new ArrayList<>();
 
-        for (Map.Entry<String, BundleData> entry : loadData.getBundleData().entrySet()) {
+        int updateBundleCount = selectTopKBundle().join();
```

Comment:
Should we control this topk bundle logic under a knob?

Reply:
Good idea, i push a commit to disable it if the `loadBalancerMaxNumberOfBundlesInBundleLoadReport <= 0`.

Reply:
Please remember to update the `loadBalancerMaxNumberOfBundlesInBundleLoadReport` configuration document.

https://github.com/apache/pulsar/blob/949260f190c3ff48d16f9450083c2e8c5c9ff302/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2658-L2667

Reply:
updated, thanks.